docs/enriching-your-data/available-enrichments/custom-javascript-enrichment/index.md

```diff
@@ -131,6 +139,117 @@ Manipulating your event stream with Javascript is powerful, but can be dangerous
 - include CPU-intensive tasks without being aware of the impact on your event processing time
 - allow untrusted parties to write your script – the script has access to the Java standard library and therefore to your filesystem.
 
+## Testing your enrichment
+
+You can test your enrichment with [Snowplow Micro](/docs/getting-started-with-micro/what-is-micro/index.md) before adding it to your production pipeline. Follow the Micro [usage guide](/docs/getting-started-with-micro/configuring-enrichments/index.md) to set up Micro and configure it to use your enrichment.
+
+:::tip Testing `.js` files directly
+
+One trick up Micro’s sleeve is that while you can add the full enrichment configuration in a JSON file (with your JavaScript encoded in base64), you can also just drop a `.js` file in your enrichments folder. Micro will interpret it as JavaScript enrichment code. This way, you don’t need to keep converting your enrichment to base64 as you iterate on it. _(Make sure to either use a `.json` or a `.js` file, but not both.)_
+
+:::
+
+Let’s walk through an example.
+
+### Step 1. Write your enrichment
+
+Suppose you want to add avocados to all your events:
+
+```js title="avocado.js"
+function process(event) {
+    return [{
+        schema: "iglu:com.snowplowanalytics.snowplow/add_to_cart/jsonschema/1-0-0",
+        data:  { sku: "avocado", quantity: "a lot" }
+    }]
+}
+```
```

Comment:
Can we make a more snowplow-themed joke here instead?

Reply:
You mean like an IKEA joke?